### PR TITLE
Wrote Die class

### PR DESCRIPTION
**Pull request by kernohad**

### CODE REVIEW

File: Model/src/GamePackage/Die.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ *****************************************************************/
+public class Die {
+
+    /** current value of the die */
+    int value;
+
+    /*****************************************************************
+     Default constructor
+     *****************************************************************/
+    public Die( ) {
```
Comment (quirogas) [inline Model/src/GamePackage/Die.java]: The constructor should instantiate the value . Any value you chose between 1- 6 should be fine.

File: Model/src/GamePackage/Die.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package GamePackage;
+
+import java.util.Random;
+
+/*****************************************************************
+ The Die Class will be a random number generator that "rolls"
+ a die some methods to return the value of the die.
+
+ @author Dylan Kernohan
+ @version Winter 2018
```
Comment (quirogas) [inline Model/src/GamePackage/Die.java]: The version should have the whole date

File: Model/src/GamePackage/Die.java (hunk 1 above)
Comment (kernohad) [reply Model/src/GamePackage/Die.java]: We do not want the constructor to instantiate a value. When we create the dice, it does not need a value until we call role() honestly could get rid of constructor, but its good practice to write it in, even if empty.

File: Model/src/GamePackage/Die.java (hunk 2 above)
Comment (kernohad) [reply Model/src/GamePackage/Die.java]: I was going off the style guide, their example had it like this. We can change it though, not a big deal.

**Pull request merged**